0-email-signup-form/FCCvote-net-neutrality-thanks/below-form-content.html

```diff
@@ -0,0 +1,72 @@
+<!-- =================== Below form content STARTS =================== -->
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```

Comment:
Does parsley work with the *ancient* version of jquery that BSD includes by default? (if so, it would lighten the page load a ton to drop 2.1.1)